specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          }
+        }
+      }
+    },
+    "/livy/batches/{batchId}/log": {
+      "get": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Gets a single spark batch job logs.",
+        "operationId": "SparkBatch_GetLogs",
+        "x-ms-examples": {
+          "Gets a single spark batch job logs": {
+            "$ref": "./examples/HDI_SparkBatch_GetLogs.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "batchId",
+            "in": "path",
+            "description": "Identifier for the batch job.",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "from",
+            "in": "query",
+            "description": "Optional param specifying which index the list should begin from.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "size",
+            "in": "query",
+            "description": "Optional param specifying the size of the returned list.\r\n            By default it is 20 and that is the maximum.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkJobLog"
+            }
+          }
+        }
+      }
+    },
+    "/livy/batches/{batchId}/state": {
+      "get": {
+        "tags": [
+          "SparkBatch"
+        ],
+        "description": "Gets a single spark batch state.",
+        "operationId": "SparkBatch_GetState",
+        "x-ms-examples": {
+          "Gets a single spark batch state": {
+            "$ref": "./examples/HDI_SparkBatch_GetState.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "batchId",
+            "in": "path",
+            "description": "Identifier for the batch job.",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkJobState"
+            }
+          }
+        }
+      }
+    },
+    "/livy/sessions": {
+      "get": {
+        "tags": [
+          "SparkSession"
+        ],
+        "description": "List all spark sessions.",
+        "operationId": "SparkSession_List",
```

Comment:
Convention for operation group (the first part before the _) is to use plural form:
`SparkSessions_List`

Please use this convention for the entire file

Reply:
Got it. I will update all operationIds using above convention.